FileChooser.py

```diff
@@ -32,26 +54,33 @@ def getMediaPath(fileName):
     Returns:
         the full path for the file
     """
-    return os.path.join(getMediaDirectory(), fileName)
-    #return os.path.join(JESConfig.CONFIG_MEDIAPATH, fileName)
+    return os.path.join(JESConfig.getConfigVal("CONFIG_MEDIAPATH"), fileName)
 
 def getMediaDirectory():
     """Method to get the directory for the media
 
     Returns:
         the media directory
     """
-    return JESConfig.CONFIG_MEDIAPATH
+    return JESConfig.getConfigVal("CONFIG_MEDIAPATH")
 
 def setMediaPath(directory):
     """Method to set the media path by setting the directory to use
 
     Parameters:
         directory - the directory to use for the media path
     """
-    JESConfig.CONFIG_MEDIAPATH = directory
+    JESConfig.setConfigVal("CONFIG_MEDIAPATH", directory)
 
 def pickMediaPath():
-    """Method to pick a media path using the file chooser and set it
-    """
-    JESConfig.CONFIG_MEDIAPATH = eg.diropenbox(title="Choose Media Path")
+    app = wx.App()
```

Comment:
Why not just call pickADirectory() and then use the returned path to set as the media path?